core/src/main/java/com/linecorp/armeria/common/logging/RequestLog.java

```diff
@@ -67,11 +65,6 @@
  */
 public interface RequestLog extends AttributeMap {
```

Comment:
A thought for another PR - I wonder if it would simplify things to make `RequestLog` not be `AttributeMap`. Since `RequestLog.context()` always gives an attribute map, I'm confused between the relationship, and I guess it could provide some improved efficiency to not have two attribute maps that essentially represent a single concept.

Reply:
Just a shortcut for `log.context().attr()`? Hm, 🤔 not sure which one is best. 🤣

Reply:
Yeah, they are just a shortcut. They refer to the same map.

Reply:
Ah didn't notice it's just a shortcut :) I think it might be better not to extend `AttributeMap` and if the methods seem useful (quite a small shortcut...) define them directly with that explanation. I expected something that implements `AttributeMap` to have a map of attributes itself